### ISSUE TITLE
0.4.3.88 - check_pagefile - Incoherent values of pagefile

### ISSUE THREAD

**Issue by quentinou** [mickem/nscp#96]
Hi Michael,

I don't understand the values from 'check_pagefile'.
First, the command:
```bash
# /usr/lib/nagios/plugins/check_nrpe -H xxxxxx -p 5666 -t 30 -u -c check_pagefile -a 'detail-syntax=${name} ${used} (${size})' "warning=" "critical=" "perf-config=used(unit:B)" "filter=" show-all
OK: \??\G:\pagefile.sys 1.115GB (1.268GB), total 1.115GB (1.268GB)|
```

I have a value of 1,2GB. Whereas, that's not the same value (649) on Windows 2003 R2 SP2 (same on 2008 R2). I have put some attachments.

Thanks for your work.
![screen1](https://cloud.githubusercontent.com/assets/9079781/5956490/be44e832-a7ae-11e4-955c-aae0433d50e1.png)
![screen2](https://cloud.githubusercontent.com/assets/9079781/5956491/be460e06-a7ae-11e4-8b38-228860ef591a.png)

**Comment by mickem** [comment]
Sorry the pagesize was a bit messed up...
This will be fixed in the next build.

**Comment by mickem** [comment]
This should be fixed in nightly build 0.4.3.104.
